go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/omec-project/config5g v1.5.5
 	github.com/omec-project/openapi v1.3.2
-	github.com/omec-project/util v1.2.8
+	github.com/omec-project/util v1.2.10
```

Comment:
I think you can use `util v1.2.11` (when resolving the conflicts in go.mod and go.sum)